io.catenax.sealant.sealant_pass/1.0.0/SealantPass.ttl

```diff
@@ -0,0 +1,648 @@
+#######################################################################
+# Copyright (c) 2023 BASF SE
+# Copyright (c) 2023 Robert Bosch GmbH
+# Copyright (c) 2023 Henkel AG & Co. KGaA
+# Copyright (c) 2023 SAP SE
+# Copyright (c) 2023 T-Systems International GmbH
+# Copyright (c) 2023 ZF Friedrichshafen AG
+# Copyright (c) 2023 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This work is made available under the terms of the 
+# Creative Commons Attribution 4.0 International (CC-BY-4.0) license, 
+# which is available at
+# https://creativecommons.org/licenses/by/4.0/legalcode.
+#
+# SPDX-License-Identifier: CC-BY-4.0
+#######################################################################
+
+
+@prefix bamm: <urn:bamm:io.openmanufacturing:meta-model:1.0.0#>.
+@prefix bamm-c: <urn:bamm:io.openmanufacturing:characteristic:1.0.0#>.
+@prefix bamm-e: <urn:bamm:io.openmanufacturing:entity:1.0.0#>.
+@prefix unit: <urn:bamm:io.openmanufacturing:unit:1.0.0#>.
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
+@prefix : <urn:bamm:io.catenax.sealant.sealant_pass:1.0.0#>.
+@prefix address-c: <urn:bamm:io.catenax.shared.address_characteristic:1.0.1#>.
+
+
+:SealantPass a bamm:Aspect;
+    bamm:properties (:documents :performanceProperties :serialNumber :productIdentification :application :supplier :hazardsIdentification :productComposition :safetyMeasures :productProperties :productDisposal :transportInformation :regulatoryInformation :carbonFootprint);
+    bamm:operations ();
+    bamm:events ();
+    bamm:preferredName "sealant pass"@en;
+    bamm:description "The Sealant Passport represents the needs from manufacturers of chemical subtance and mixtures (e.g. BASF, Henkel). It describes data required by the safety data sheet - which is based on the REACH Regulation No 1907/2006 in the format of a DPP."@en;
```

Comment:
not sure whether real company names should be included in model.

Reply:
done